src/components/calculator/SavedDataSelector.tsx

```diff
@@ -47,6 +47,7 @@ export const SavedDataSelector: React.FunctionComponent<{
           Optional: Start with a predefined common activity...
         </option>
         {prepopulatedOptions}
+        <optgroup label=""></optgroup>
```

Comment:
Could you add comments to explain the purpose of these? I found one the other day and removed it because I thought it was a bug.

Reply:
ahahahaha. Oopsies.

Reply:
Did one better - its now in a documented component that only renders on mobile ios.